src/components/Button/index.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import cx from 'classnames';
+import Spinner from '../Spinner';
+import Icon from '../Icon';
+
+const INTENTS = {
+  PRIMARY: 'primary',
+  SECONDARY: 'secondary',
+};
+
+const VARIANTS = {
+  OUTLINE: 'outline',
+  MINIMAL: 'minimal',
+};
+
+const intents = Object.values(INTENTS);
+const variants = Object.values(VARIANTS);
+
+const styles = {
+  color({ disabled, variant, intent }) {
+    if (disabled) {
+      return 'var(--f-color-text--tertiary) !important';
```

Comment:
Do we plan to have a theme in JS at some point?

Reply:
I was thinking about this yesterday, make it both a standalone exported object and a SC theme, so that we can reuse it easily. I'm getting fed up of not remembering the name of the CSS vars 😅.

I can do that in another PR

Reply:
Really like the idea of having it as a standalone exported object